hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,160 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_aes
+  testpoints: [
+    {
+      name: chip_sw_aes_enc
+      desc: '''Verify the AES operation.
+
+            Write a 32-byte key and a 16-byte plain text to the AES registers and trigger the AES
+            computation to start. Wait for the AES operation to complete by polling the status
+            register. Check the digest registers for correctness against the expected digest value.
+            Repeat the test for all three supported key lengths (128/192/156).
+            '''
+      features: [
+        "AES.MODE.ECB",
+      ]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
```

Comment:
Can you explain what the motivation is to run this test in all these LC states? Do we really need this?